package.json

```diff
@@ -15,6 +15,10 @@
   "license": "MIT",
```

Comment:
https://github.com/OSCA-Kampala-Chapter/react-ui/blob/9f0babe7cb83754d1d2e6080e0096ec48a25fa57/package.json#L6
under `scripts` please add a trigger for the linting
```sh
   "lint:js": "eslint --cache \"**/*.{js,jsx,ts,tsx,md,mdx,mjs}\"",
   "lint:js:fix": "eslint \"**/*.{js,jsx,ts,tsx,md,mdx,mjs}\" --fix",
```
whatever is in the object `{js,jsx,ts,tsx,md,mdx,mjs}` are file types we want to look out for to lint

References incase you wish to look at tags
https://eslint.org/docs/latest/use/command-line-interface#options